webpack.common.js

```diff
@@ -16,6 +16,7 @@ const Assets = [
 module.exports = {
     context: path.resolve(__dirname, 'src'),
     entry: './bundle.js',
+    stats: 'errors-only',
```

Comment:
What's up with this wouldn't it be better to set that by cli if needed

Reply:
It's meant to hide all the non essential output of webpack (stats about bundle size and so on) during builds